src/main/webapp/scripts/appscript.js

```diff
@@ -79,11 +79,21 @@ function buildEditForm() {
 function buildQueryString() {
   const selectedVendorId = document.getElementById('customer-ids').value;
   const selectedAccountId = document.getElementById('account-ids').value;
-  return `/BillingConfig?vendorID=${selectedVendorId}&accountID=${selectedAccountId}`;
+  // check for space in accountID.
+  if(selectedAccountId == "") {
+    window.alert("A vendor ID and account ID have not been set yet!");
+  } else {
+    if(document.getElementById('add-account') != null) {
+      document.getElementById('add-account').style.display = 'block';
```

Comment:
`buildQueryString` does not seem like the appropriate place to be making changes to CSS styling. Please move this to another method